cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -167,7 +187,9 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
   @Test def drop_index() {
     prepareAndTestQuery(
       title = "Drop an index",
-      text = "An index on all nodes that have a label and property/properties combination can be dropped using the name with the `DROP INDEX index_name` command.",
+      text =
+        """An index on all nodes that have a label and property/properties combination can be dropped using the name with the `DROP INDEX index_name` command.
+          |The name of the index can be found using <<administration-indexes-list-indexes, `SHOW INDEXES`>>, given in the output column `name`.""".stripMargin,
```

Comment:
It is not easy to understand that `SHOW INDEXES` is a link here given that it is back-ticked, but maybe that is different in the new design? One option is to change it to 

the <<administration-indexes-list-indexes, `SHOW INDEX` command>> 

to make it consistent with the `DROP INDEX` link in the `SHOW INDEX` example .